NASB_Parser/StateActions/SAConfigHitbox.cs

```diff
@@ -52,7 +52,7 @@ public override void Write(BulkSerializeWriter writer)
             writer.Write(TID);
             writer.Write(1);
             writer.Write(Hitbox);
-            writer.Write(ForceZ0);
```

Comment:
What changed here, are you sure this is correct?

Reply:
Yeah, between the two versions that I compared.
Here is a screenshot from the repo I'm using to compare.
The latest update, 21.5.0, is on the right.

![image](https://user-images.githubusercontent.com/27714637/145506124-ab2d739f-fd4a-421e-a08a-09408cdffd05.png)

Reply:
The other change in that file is simply hiding the toggle for that boolean in their "EditDraw" function, so I assume it's an unused value now.
![image](https://user-images.githubusercontent.com/27714637/145506304-a6f3b460-0623-403a-a633-39d6dae065b5.png)